src/wp-includes/html-api/class-wp-html-active-formatting-elements.php

```diff
@@ -43,6 +43,20 @@ class WP_HTML_Active_Formatting_Elements {
 	 */
 	private $stack = array();
 
+	/**
+	 * Returns the node at the given index in the list of active formatting elements.
+	 *
+	 * Do not use this method; it is meant to be used only by the HTML Processor.
+	 *
+	 * @access private
+	 *
+	 * @param int $index Number of nodes from the top node to return.
+	 * @return WP_HTML_Token|null Node at the given index in the stack, if one exists, otherwise null.
+	 */
+	public function at( $index ) {
```

Comment:
Maybe some private looking prefix is a good idea to suggest "this isn't something that should be used publicly":

```suggestion
	public function _at( $index ) {
```

Reply:
this is what `@access private` is for. we shouldn't need to apply any other mechanisms.